.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,69 @@
+name: Maven e2e test - simple # TODO: Remove name
+on:
+  - workflow_dispatch
+# TODO: Un-comment this     
+#  schedule:
+#    - cron: "0 6 * * *"
+#  workflow_dispatch:
+#
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  bootstrap:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    permissions:
+      contents: write
+    outputs:
+      artifact-version: ${{ steps.maven-push.outputs.artifact-version }}
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - env:
+          PACKAGE_DIR: ./e2e/maven
+        id: maven-push
+        run: ./.github/workflows/scripts/e2e-maven-push.sh
+
+  if-bootstrap-failed:
+    runs-on: ubuntu-latest
+    needs: [bootstrap]
+    if: always() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && needs.bootstrap.result != 'success'
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - run: ./.github/workflows/scripts/e2e-report-failure.sh
+
+  build:
+    #runs-on: ubuntu-latest
+    permissions:
+      id-token: write # For signing.
+      contents: read # For repo checkout of private repos.
+      actions: read # For getting workflow run on private repos.
+    uses: AdamKorcz/slsa-github-generator/.github/workflows/builder_maven_slsa3.yml@maven-builder-test-updates
+    with:
+      directory: ./e2e/maven
+
+  verify:
+    runs-on: ubuntu-latest
+    needs: [build, bootstrap]
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: slsa-framework/slsa-github-generator/.github/actions/secure-download-folder@main
+        with:
+          name: "${{ needs.build.outputs.provenance-download-name }}"
+          sha256: "${{ needs.build.outputs.provenance-download-sha256 }}"
+          path: slsa-attestations
+      - uses: slsa-framework/slsa-github-generator/.github/actions/secure-download-folder@main
+        with:
+          name: target
+          sha256: "${{ needs.build.outputs.target-download-sha256 }}"
+          path: ./
+      - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
+        with:
+          go-version: "1.18"
+      - env:
+          ARTIFACT_VERSION: ${{ needs.bootstrap.outputs.artifact-version}}
+          PROVENANCE: "slsa-attestations/${{ needs.build.outputs.provenance-download-name }}/test-java-project-${{ needs.bootstrap.outputs.artifact-version}}.jar.build.slsa"
+          BUILDER_TAG: "v2.0.0"
```

Comment:
you're missing the failure / success jobs https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.nodejs.workflow_dispatch.main.default.slsa3.yml#L107-L121
These will create issues on the main slsa-github-generator repo if things fails (and close them if they succeed)